README.md

```diff
@@ -9,23 +9,40 @@ using [Elasticsearch](https://www.elastic.co/).
 Increase the maximum number of virtual memory areas on your host
 system. On a Linux system this can be done using the command line
 
-    sudo sysctl -w vm.max_map_count=262144
+    sysctl -w vm.max_map_count=262144
```

Comment:
Are you sure about this ? I mean does changing system config as normal user always work ?

Reply:
It does have to be as root, but for some reason I have an aversion to including `sudo` in tutorials. Maybe I should get over this.. I liked how the ES docs put it: "now run the following command as root": `sysctl -w vm.max_map_count=262144`

Thoughts?

https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html

Reply:
I also don't like when sudo is in docs. I you don't know when to use sudo, you need to better understand the possibly system breaking commands you are about to run.

Reply:
Either is fine - explicitly state that the following command must be run as root or add `sudo` to the command prefix. Take your pick.